src/LinAlg/hiopVectorPar.hpp

```diff
@@ -70,6 +70,7 @@ class hiopVectorPar : public hiopVector
 
   virtual void setToZero();
   virtual void setToConstant( double c );
+  virtual void set_to_random_constant(double minv, double maxv);
```

Comment:
"constant" is a bit confusing maybe? name need to reflect that the uniform distribution is sampled. Did you mean `set_to_random_uniform` ?